src/main/scala/viper/silver/reporter/Message.scala

```diff
@@ -74,26 +75,24 @@ case class OverallFailureMessage(verifier: String, verificationTime: Time, resul
 }
 
 // Entity results concern results for specific program entities (these are presently produced by the Silicon backend)
-case class EntitySuccessMessage(verifier: String, concerning: Entity, verificationTime: Time)
-    extends VerificationResultMessage {
+case class EntitySuccessMessage(verifier: String, concerning: Entity, verificationTime: Time, cached:Boolean = false, result: VerificationResult = Success)
```

Comment:
Why do we need a result field for EntitySuccessMessage? The super type, VerificationResultMessage, does not seem to require this.

Reply:
But there's a result field in VerificaitonResultMessage?

https://github.com/viperproject/silver/blob/8c6b539d1a333d90e5e534eba2050618d27a2fc3/src/main/scala/viper/silver/reporter/Message.scala#L25